src/main/controllers/CitizenHubController.ts

```diff
@@ -40,6 +41,7 @@ export default class CitizenHubController {
     }
 
     const userCase = req.session.userCase;
+    clearTseFields(userCase);
```

Comment:
Why do you need to clear Tse fields here?

Reply:
Because on CYA page you can press cancel and will be redirected back to citizen hub

Scenario 7 – Claimant can cancel the process

Given the Application CYA screen is displayed

When claimant selects ‘Cancel’

Then system should cancel the process (and any saved data)

And display the CitizenUI case overview/hub page ([RET-2260](https://tools.hmcts.net/jira/browse/RET-2260))